src/osemgrep/cli_install/Install_subcommand.ml

```diff
@@ -397,7 +404,8 @@ let add_semgrep_workflow ~repo ~token ~update : unit =
 (*****************************************************************************)
 
 let run (conf : Install_CLI.conf) : Exit_code.t =
-  CLI_common.setup_logging ~force_color:true ~level:conf.commons.logging_level;
+  CLI_common.setup_logging ~force_color:true ~level:conf.common.logging_level;
+  Metrics_.configure Metrics_.Auto;
```

Comment:
maybe we should accept the --metrics option here?
or maybe just add a comment saying it does not make sense to let the user do --metrics=off
for install-ci so we set it to Auto by default.